src/frontend/pages/enrich.vue

```diff
@@ -0,0 +1,373 @@
+          />
+
+          <v-text-field
+            v-model="extensionSelected"
+            :label="$t('enrich.fileExtension')"
+            filled
+            readonly
+          />
+
+          <v-toolbar class="secondary" dark dense flat>
+            <v-toolbar-title v-text="$t('enrich.unpaywallAttributes')" />
+          </v-toolbar>
+
+          <SettingsCSV v-if="extensionSelected === 'csv'" />
+          <SettingsJSONL v-if="extensionSelected === 'jsonl'" />
+        </v-stepper-content>
+
+        <v-stepper-content step="3">
+          <v-container>
+            <v-layout row justify-end class="mb-3">
+              <v-btn :href="resultUrl" :disabled="inProcess || error">
+                <v-icon left>
+                  mdi-download
+                </v-icon>
+                {{ $t("enrich.download") }}
+              </v-btn>
+            </v-layout>
+          </v-container>
+          <v-container v-if="inProcess" class="text-center">
+            <v-progress-circular
+              :size="70"
+              :width="7"
+              indeterminate
+              color="green"
+            />
+            <div v-text="$t('enrich.inProcess')" />
+          </v-container>
+          <v-container v-else class="text-center">
+            <v-icon v-if="error" size="70" color="orange darken-2">
+              mdi-alert-circle
+            </v-icon>
+            <v-icon v-else size="70" color="green darken-2">
+              mdi-check
+            </v-icon>
+            <div v-if="error" v-text="$t('enrich.error')" />
+            <div v-else v-text="$t('enrich.end')" />
+          </v-container>
+          <v-container>
+            <Report :time="time" :state="state" />
+          </v-container>
+        </v-stepper-content>
+      </v-stepper-items>
+    </v-stepper>
+  </v-card>
+</template>
+
+<script>
+import LogFiles from '~/components/enrich/LogFiles.vue'
+import SettingsCSV from '~/components/enrich/SettingsCSV.vue'
+import SettingsJSONL from '~/components/enrich/SettingsJSONL.vue'
+import Report from '~/components/enrich/Report.vue'
+
+export default {
+  name: 'Enrich',
+  components: {
+    LogFiles,
+    SettingsCSV,
+    SettingsJSONL,
+    Report
+  },
+  transition: 'slide-x-transition',
+  data: () => {
+    return {
+      // stepper
+      step: 1,
+      // config
+      files: [],
+      apiKey: '',
+      apiKeyVisible: false,
+      apiKeyRules: {
+        required: value => !!value || 'Required.'
+      },
+      // help
+      fileSelectionHelp: false,
+      logSamplesUrl: 'https://github.com/ezpaarse-project/ezunpaywall',
+      // process
+      state: {},
+      time: 0,
+      inProcess: false,
+      error: false,
+      id: ''
+    }
+  },
+  computed: {
+    // config
+    hasLogFiles () {
+      return Array.isArray(this.files) && this.files.length > 0
+    },
+    extensionSelected () {
+      if (this.files.length !== 0) {
+        const [, ext] = this.files[0].file.name.split('.')
+        return ext
```

Comment:
```suggestion
        return this.files[0]?.file?.name?.split?('.')
```
Won't work for files with multiple dots in their name